tests/integration/annotation_import/test_model_run.py

```diff
@@ -114,6 +114,15 @@ def test_model_run_export_labels(model_run_with_model_run_data_rows):
     assert len(labels) == 3
 
 
+def test_model_run_export_labels_v2(model_run_with_model_run_data_rows):
+    task_name = "test_task"
+    task = model_run_with_model_run_data_rows.export_labels_v2(
+        task_name, filter={"media_attributes: true"})
+    assert task.name == task_name
+    task.wait_till_done()
+    assert task.status == "COMPLETE"
```

Comment:
It would be good to check at least that the high level keys exist in the payload and the number of items returned was expected.

Reply:
Shouldn't that be in the scope of integration tests of API, not the SDK?

Reply:
The SDK is the most useful end to end integration test. So it is nice to have it here to guarantee that things are working (also it blocks deployments). But technically it is weird that it serves that purpose.